tests/console/yast2_apparmor.pm

```diff
@@ -33,8 +33,10 @@ sub run {
         send_key 'alt-e';
     }
     assert_screen 'yast2_apparmor_enabled';
+
     # part 1: open profile mode configuration and check toggle/show all profiles
-    send_key 'alt-n';
+    send_key((is_pre_15 || is_tumbleweed) ? 'alt-n' : 'alt-c');
```

Comment:
no, wait. Something looks fishy now. tw should behave like 15 so to my understanding it should be

```perl
send_key is_pre_15? 'alt-n' : 'alt-c';
```

Reply:
You're right, I added TW to the is_pre_15 functionality